picard_plugin_tools/__init__.py

```diff
@@ -246,19 +231,26 @@ def package_folder(plugin_dir, manifest_path, output_path=None):
         archive.write(manifest_path,
                       'MANIFEST.json',
                       compress_type=zipfile.ZIP_DEFLATED)
-    info_list = archive.infolist()
+    return manifest_data
 
 
-@cli.command()
-@click.argument('archive_path', type=click.Path(exists=True))
-def verify_package(archive_path):
-    """Verifies the checksum of a packaged plugin and verifies its
-        integrity
+@cli.command('package_folder')
+@click.argument('plugin_dir', type=click.Path(exists=True))
```

Comment:
Instead of `argument` which is positional i would use `option`: http://click.pocoo.org/5/options/

The main idea is to have:

`program command --command-option1 param1 --command-option2 param2 ... `

Using long options makes the command line self-documented, it is longer to type (than `program command param1 param2`) but much easier to understand without looking at the doc.